Task1/aga/IsPrime/IsPrime/IsPrime.cs

```diff
@@ -0,0 +1,54 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace IsPrime
+{
+    class IsPrime
```

Comment:
Name is confusing - IsPrime suggest predicate that is capable of answering such question. Not a sieve implementation.